### PR TITLE
Add docs for backing up the wallet

### LINKED ISSUE #3322: Add documentation for backing up DLC wallet
A question from a user 
Screenshot from 2021-06-23 13-11-10

we should have a page on the website we can easily link to so they can understand what is backed up and how to restore

### PR DESCRIPTION
**Pull request by benthecarman**
Closes #3322

### CODE REVIEW

File: docs/wallet/backups.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+archive of the `.bitcoin-s` data folder. This is chosen because to have a complete backup
+for open DLCs you need to restore all the data given from your counter-party as well as some
+generated by your own wallet. Zipping the folder is the easiest way to back up and restore
+this data as it is not something that can be written down on a piece of paper.
+
+## Backing up from the GUI
+
+To create a backup from the GUI simply click the `File` menu, then select `Save Backup`.
+This will bring up a dialog to choose the save location for where to save the file to.
+
```
Comment (Christewart) [inline docs/wallet/backups.md:16]: nit: maybe add image here?

**Review by Christewart — COMMENTED** [review]
LGTM but also think it would be a useful test/exercise for @user411 to run through

File: docs/wallet/backups.md (hunk 1 above)
Comment (benthecarman) [reply docs/wallet/backups.md:16]: Not sure how to do that with our docs

Comment (Christewart) [reply docs/wallet/backups.md:16]: Take a look at #3299

Comment (Christewart) [reply docs/wallet/backups.md:16]: Specifically here: https://github.com/bitcoin-s/bitcoin-s/pull/3299/files#diff-31bcba2ccafa41d46fbbd6d1219f7f1e3b1fb3cad9faa8e4dc521bbb579dd7b3R38

Comment (Christewart) [reply docs/wallet/backups.md:16]: nit: maybe we should add a `docs/images/` directory too

Comment (benthecarman) [reply docs/wallet/backups.md:16]: Added, also fixed the image of the one you added as it was not being displayed

**Review by Christewart — APPROVED** [review]
LGTM, waiting for @user411 for confirmation on it working

**Review by user411 — APPROVED** [review]
Output in backup file look as expected vs .bitcoin-s contents.

**Pull request merged**